src/layouts/BlogPostLayout.astro

```diff
@@ -42,6 +42,18 @@ import "../styles/global.css";
           <h1 class="text-3xl">{frontmatter.title}</h1>
         </header>
         <slot />
+        <style is:global>
+          h3, p {
+             padding-bottom: 10px;
+          }
+          a {
+            text-decoration: underline;
+          }
+          a:hover {
+            text-decoration: none;
+          }
```

Comment:
This is probably nicer than what we have for our footer links. (Footer links are the reverse, but this doesn't work on mobile). 

With that said, what about having a global link style?
https://github.com/Lobstars-BVB/Lobstars/blob/main/src/styles/global.css